js/apps/admin-ui/src/clients/authorization/AuthorizationEvaluate.tsx

```diff
@@ -285,26 +279,12 @@ const AuthorizationEvaluateContent = ({ client }: Props) => {
           </PanelHeader>
           <PanelMainBody>
             <FormAccess isHorizontal role="view-clients">
-              <FormGroup
+              <DefaultSwitchControl
+                name="applyToResourceType"
                 label={t("applyToResourceType")}
-                fieldId="applyToResourceType"
-                labelIcon={
-                  <HelpItem
-                    helpText={t("applyToResourceTypeHelp")}
-                    fieldLabelId="applyToResourceType"
-                  />
-                }
-              >
-                <Switch
-                  id="applyToResource-switch"
-                  label={t("on")}
-                  labelOff={t("off")}
-                  isChecked={applyToResourceType}
-                  onChange={setApplyToResourceType}
-                  aria-label={t("applyToResourceType")}
-                />
-              </FormGroup>
-
+                labelIcon={t("applyToResourceTypeHelp")}
+                onChange={setApplyToResourceType}
+              />
```

Comment:
this might not work as it now also saves this setting in the form

Reply:
Right, I'm going to revert this change and use original Switch implementation instead.